presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -100,26 +101,28 @@ Date and Time Functions
         SELECT from_iso8601_date('2020-123');
         -- 2020-05-02
 
-.. function:: at_timezone(timestamp, zone) -> timestamp with time zone
+.. function:: at_timezone(timestamp(p), zone) -> timestamp(p) with time zone
 
-    Change the time zone component of ``timestamp`` to ``zone`` while preserving the instant in time.
+    Change the time zone component of ``timestamp`` with precision ``p`` to
+    ``zone`` while preserving the instant in time.
 
-.. function:: with_timezone(timestamp, zone) -> timestamp with time zone
+.. function:: with_timezone(timestamp(p), zone) -> timestamp(p) with time zone
```

Comment:
Same as `at_timezone`, only change the return type, not the argument name or description.

Reply:
I have done that now in both cases but the above concerns about it not being clear what timestamp data type is supported as input parameter remains.